keyboards/redox/keymaps/finex/keymap.c

```diff
@@ -3,47 +3,159 @@
 extern keymap_config_t keymap_config;
 
 // Layer names
-#define _QWERTY 0
-#define _COLEMAKDHMK 1
-#define _QWERTY_MAC 2
-#define _COLEMAKDHMK_MAC 3
+// - Base layers:
+#define _COLEMAKDHMK 0
```

Comment:
You may want to consider using an `enum` here, as it may be easier to manage in the future.

Reply:
Thank you @drashna for the suggestions. I've used an enum and committed to my master branch (next time I will create a branch). Tell me if it is ok :-)

Reply:
Yup, that's great